_posts/2018-10-19-devops-conference.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: DevOps Conference Learnings
+author: jake_hall
+---
+
+I recently attended the National DevOps Conference. I sometimes find it useful to make notes during these kinds of events, and rather than them being buried, unloved, somewhere hidden. I've written them up, so here they are for the world to see, and hopefully you'll find them useful.
+
+# What is it?
+
+The National DevOps Conference is a 2 day conference, hosted at the Millennium Gloucester Hotel in Kensington. It has 3 tracks of talks on at any given time, bookended by keynote speakers.
+
+# What did I learn?
+
+## Kotter's Model/Cycle
+
+If you want to implement change in an organisation, something we regularly do as consultants, you'll probably need this in your back pocket. The most important part of starting a change is to **create urgency** or a need for the change, otherwise it will be less likely to happen.
+
+1. Create: A need for change and urgency
+2. Build: An internal team to manage the change
+3. Form: Make a strategy which aligns with business vision
+4. Enlist: Encourage volunteers and engage them in meaningful activities to support the change
+5. Enable: Reduce the ineffective and unnecessary processes from the system
+6. Generate: Set short-term goals leading towards the long-term change
+7. Sustain: Moreover manage planning, finances and processes with change leadership
+8. Institute: Lastly, communicating the importance of keeping up with change to all employees and show transparency in the processes.
+
+Thanks to Keith Watson from ADP for this one!
+
+## Intra-cluster security
+
+We all know that securing our services from the internet, or North/South traffic is absolutely paramount. Whether this is locking down firewalls, or security groups, to using internal only IPs. There are many, many options whether you're on-premise or in cloud.
+
+What is becoming apparent is the need to protect our East/West traffic, that is our intra-cluster traffic. The traffic when each of our services talks to another one of our services.
+
+One of the ways to protect and secure our East/West traffic is to use a Service Mesh. [Istio](https://istio.io/) was recommended by David Luke as an excellent option.
```

Comment:
Worth mentioning that Istio is Kubernetes only (you mention cluster security, but it sounds like it's actually just talking about Kube?). Also Istio gives you mutual TLS basically for free ❤️